### PR TITLE
blob typo fixes

### PR DESCRIPTION
**Pull request by lanse12**
## About the PR
fixes typos related to blob shit

### CODE REVIEW

**Comment by JoeHammad1844** [comment]
the emergency shuttle shouldnt be called, anthougheversobeit

File: Resources/Locale/en-US/_Goobstation/blob/blob.ftl [hunk 1]
```diff
@@ -89,7 +89,7 @@ blob-alert-out-off-station = The blob was removed because it was found outside t
 
 # Announcment
 blob-alert-recall-shuttle = The emergency shuttle can not be sent while there is a level 5 biohazard present on the station.
-blob-alert-detect = Confirmed outbreak of level 5 biohazard aboard the station. All personnel must contain the outbreak. The emergency shuttles can not be sent due to contamination risks.
+blob-alert-detect = Confirmed outbreak of level 5 biohazard aboard the station. All personnel must contain the outbreak.
```
Comment (Piras314) [inline Resources/Locale/en-US/_Goobstation/blob/blob.ftl:92]: keep the old line here

File: Resources/Locale/en-US/_Goobstation/blob/blob.ftl [hunk 2]
```diff
@@ -152,7 +152,7 @@ blob-zombie-greeting = You were infected and raised by a blob spore. Now you mus
 # End round
 blob-round-end-result =
     { $blobCount ->
-        [one] There was one blob.
+        [one] There was one blob infection.
```
Comment (Piras314) [inline Resources/Locale/en-US/_Goobstation/blob/blob.ftl:155]: keep the old line here

File: Resources/Locale/en-US/_Goobstation/blob/blob.ftl [hunk 3]
```diff
@@ -2,37 +2,37 @@ ent-SpawnPointGhostBlob = Blob spawner
...
 objective-issuer-blob = Blob
 
 
 ghost-role-information-blobbernaut-name = Blobbernaut
-ghost-role-information-blobbernaut-description = You are a Blobbernaut. You must defend blob core.
+ghost-role-information-blobbernaut-description = You are a Blobbernaut. You must defend the blob core.
 
 ghost-role-information-blob-name = Blob
-ghost-role-information-blob-description = You are a Blob. You must consume the station.
+ghost-role-information-blob-description = You are the Blob Infection. Consume the station.
```
Comment (Piras314) [inline Resources/Locale/en-US/_Goobstation/blob/blob.ftl:32]: the blob is fine, but the blob infection seems a bit silly to me

**Review by Piras314 — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/_Goobstation/blob/blob.ftl (hunk 3 above)
Comment (lanse12) [reply Resources/Locale/en-US/_Goobstation/blob/blob.ftl:32]: that's how it is on tg station bro

File: Resources/Locale/en-US/_Goobstation/blob/blob.ftl (hunk 1 above)
Comment (lanse12) [reply Resources/Locale/en-US/_Goobstation/blob/blob.ftl:92]: that's how it is on tg station bro

File: Resources/Locale/en-US/_Goobstation/blob/blob.ftl (hunk 2 above)
Comment (lanse12) [reply Resources/Locale/en-US/_Goobstation/blob/blob.ftl:155]: that's how it is on tg station bro

**Review by Piras314 — APPROVED** [review]
fair enough

**Pull request merged**